composer.json

```diff
@@ -2,19 +2,14 @@
     "name": "corneltek/pearx",
     "homepage": "http://github.com/phpbrew/PEARX",
     "description": "PEAR channel client",
-    "config": {
-        "platform": {
-            "php": "5.3.9"
```

Comment:
To resolve the following error via the `composer update corneltek/phpunit-testmore` command, it should remove config platform setting.

```
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires corneltek/phpunit-testmore dev-master -> satisfiable by corneltek/phpunit-testmore[dev-master].
    - corneltek/phpunit-testmore dev-master requires php >=8.2.3 -> your php version (7.2.5; overridden via config.platform, actual: 8.2.5) does not satisfy that requirement.
```

Reply:
Maybe we can consider adding the `--ignore-platform-reqs` option via the `composer update corneltek/phpunit-testmore` command to keep `config` setting existed.